lib/python/treadmill_aws/aws.py

```diff
@@ -1,5 +1,9 @@
 """Common AWS helper functions."""
 
+class NoInstancesReturned(Exception):
```

Comment:
we have treadmill exception resource not found ... why not use that?

is it not found, or empty list (name is confusing).

Reply:
Sure, changed to exc.NotFoundError